qa/tasks/cephfs/test_volumes.py

```diff
@@ -607,6 +613,31 @@ def test_volume_rename_for_more_than_one_data_pool(self):
         # data pool names unchanged
         self.assertCountEqual(orig_data_pool_names, list(self.fs.data_pools.values()))
 
+    def test_volume_info(self):
+        """
+        Tests the 'fs volume info' command
+        """        
+        subvol_md = ["volume_name", "gid", "size", "subvolumegroups", "pending_subvolume_deletions", "mon_addrs"]
+
+        group = self._generate_random_group_name()
+        
+        # create volume
+        self._fs_cmd("volume", "create", self.volname)
+        self._fs_cmd("subvolumegroup", "create", self.volname, group)
```

Comment:
If you are reusing `self.volname` then it's already created in the setUp().

Reply:
okay,will remove it.